.travis-ci.sh

```diff
@@ -14,6 +14,8 @@
 #*                                                                        *
 #**************************************************************************
 
+set -ex
```

Comment:
Is this an intended change or was it done for testing purpose?

Reply:
It's intended. It was originally in the `.travis.yml` (see the diff of this file) file, but it's better to have it here.